tools/azure-sdk-tools/packaging_tools/change_log.py

```diff
@@ -81,10 +92,10 @@ def models(self, diff_entry):
             return
         model_name, *remaining_path = remaining_path
         if not remaining_path:
-            # A new model or a model deletion is not very interesting by itself
-            # since it usually means that there is a new operation
-            #
-            # We might miss some discrimanator new sub-classes however
+            if is_deletion:
+                self.optional_features.append(_MODEL_REMOVE.format(model_name))
```

Comment:
only `model` removal seems not big enough to be breaking change. So treat it as features here.

Reply:
That could be debated, if I write `from azure.mgmt.mypackage.models import MyModel` and `MyModel` is removed, this would be a problem. 

Maybe we can start conservative, and just do nothing for removal (nor features, nor breaking changes, etc.) until we see a problem?

Reply:
According to release experience, only-model-removal seems never happened before. Maybe It is better to put it aside.